### ISSUE TITLE
[Project] Platform Navbar Design Enhancements

### ISSUE THREAD

**Issue by Memo-Es** [opencollective/opencollective#4356]
## Context

We have been using the same navbar for (at least) ~3.5 years now, a lot has changed ever since, and I think we should adequate a couple of things to adapt to the new reality of the platform in the present and what we want it to be in the future.

**For this series of updates, we will tackle two main goals:**
- Have an improved marketing notion from the navbar that help people to navigate between our marketing pages
- Include the 'What's new?' icon from https://github.com/opencollective/opencollective/issues/4213

**Related issues:** #4295 #4213 

@SudharakaP

**Comment by Memo-Es** [comment]
I'm presenting a proposal for the demo call of June 11th, I will update this issue with the proposal after the demo

**Comment by piamancini** [comment]
- [ ] record a loom to seek async feedback 
- [ ] prod files

**Comment by piamancini** [comment]
- [ ] record a loom and seek async feedback

**Comment by Memo-Es** [comment]
## Design update

| Description | Design proposal |
|-----------------------|-----------------|
|     **Nav bar elements new anatomy** Basically we're displaying all of our marketing elements for them to be available from the navbar |   <img width="664" alt="Screen Shot on 2022-01-24 at 18-12-53" src="https://user-images.githubusercontent.com/49539954/150832005-57af4024-fd5d-4843-88de-ad0dc25402e3.png">     |
|  **User not signed-in**      |   <img width="1290" alt="Screen Shot on 2022-01-17 at 18-41-03" src="https://user-images.githubusercontent.com/49539954/150796329-5fbc6f53-063d-4268-9e18-71ae2d6482e1.png">   |
|   **User not signed-in**   With preview of dropdown opened  |  <img width="1297" alt="Screen Shot on 2022-01-17 at 18-41-47" src="https://user-images.githubusercontent.com/49539954/150796336-197132e5-d510-411f-92a8-09cc8f4a9c3a.png">   |
|    **Signed-in user**   Here we display both kinds of notifications: Platform updates and user notifications, we're proposing to move platform updates under the Product main menu item in the navbar, so they are 2nd in importance after more user-focused ones, visible for the user attached to their avatars  |  <img width="1274" alt="Screen Shot on 2022-01-17 at 18-42-38" src="https://user-images.githubusercontent.com/49539954/150796343-14b61e2d-78d3-49d3-a631-9a2254e4fe8f.png">   |
|   **Signed-in user**   Preview of the Product dropdown to display the platform notifications by clicking on 'What's new?'    |   <img width="1292" alt="Screen Shot on 2022-01-17 at 18-49-30" src="https://user-images.githubusercontent.com/49539954/150796351-1c34bf7b-ebd6-4174-a35f-5258f26bf168.png">  |
| **Signed-in user**   Preview of the user dropdown modifications, I changed the 'View profile' button for 'Settings' and leaved your name and avatar area for going to your profile and took settings out the element list. Added the blue indicator by 'Pending invitations' so when a user is invited to be admin for a collective or is invited to submit an expense   |   <img width="1294" alt="Screen Shot on 2022-01-17 at 18-51-23" src="https://user-images.githubusercontent.com/49539954/150796358-23df47c5-76d3-4425-897d-f20ab956cbd4.png">  |

[🎬 Loom recording](https://www.loom.com/share/29de655dac7344809cd14fc738d7b564)


cc @piamancini @alanna

**Comment by alanna** [comment]
Overall this looks good! It's a lot clearer and easier to find pages.

Please ensure you're using the standard terminology in all contexts. We avoid the word "sponsors" because it's very easily confused with fiscal sponsors and many Collectives think of it as donations or memberships or whatever. The term is "financial contributors". Please capitalize Collectives, Fiscal Hosts, Financial Contributors for consistency.

I don't agree with having "Exit Strategy" in there. There's too much nuance to what we're really talking about and too much startup baggage with that terms. Let's just go with "E2C" for now and let people who are curious click to find out what it is. In the future we'll get into a proper comms strategy and naming for E2C stuff.

Could we better use the space under help & support as a dropdown menu? Docs, contact, etc.

**Comment by alanna** [comment]
Could this also solve #4761 ?

**Comment by BenJam** [comment]
just dropping a note in here that, for Q2S1 we are going to prioritise 

- [ ] Incorporating the new search capabilities
- [ ] Displaying all our marketing pages. Easier to navigate and find stuff

We'll tackle the pattern for notifications over in #4699

**Comment by Betree** [comment]
Just a warning that [we ended up adapting the menu](https://github.com/opencollective/opencollective/issues/2530) based on the page you're visiting based on [user feedback](https://github.com/opencollective/opencollective/issues/2228).

TLDR ; showing a big "Pricing" link in the top bar when you arrive at the contribution flow to make a simple contribution (without any context about what Open Collective is) might be an unnecessary distraction.

**Comment by Betree** [comment]
Some recommendations for kicking off the implementation:
- Create a separate component (e.g. `NavbarV2`) to simplify the code and make sure we're not carrying technical debt
- Ideally use `PopupMenu`, or `react-popper` to build the hoverable menus

**Comment by SudharakaP** [comment]
Thanks for the ideas @Betree. 

Some next steps for this; 

- Decide on whether we need to hide certain items in the nav bar in certain pages (such as contribution flow and discover page). This was done to give users an experience which is relevant to the context. For example if they are coming through the contribution flow they don't need to see pricing. For more context on this issue please refer to: https://github.com/opencollective/opencollective/issues/2228. 
  - For now I would suggest we keep this changing nav bar behavior for the contribution flow (we don't show pricing, how it works etc on the contribution flow). I feel we don't need to change this behavior.
  - For all other pages show the full nav bar links. 
  - Apply the new styling and links to the nav bar

**Comment by Memo-Es** [comment]
Goals for Week 2:

 - [ ] Final design input and push to implementation
 - [ ] Start implementation

**Comment by Memo-Es** [comment]
**[Implementation file](https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=16523%3A234281)** @SudharakaP

**Comment by BenJam** [comment]
- [ ] review/feedback and add mobile to https://github.com/opencollective/opencollective-frontend/pull/7758

**Comment by BenJam** [comment]
W4

- [ ] review/merge https://github.com/opencollective/opencollective-frontend/pull/7780
- [ ] take a look a look at navbar design for marketing pages (alt)

**Comment by BenJam** [comment]
- [ ] gather feedback for https://github.com/opencollective/opencollective-frontend/pull/7780 for testing

**Comment by znarf** [comment]
Merge last accessibility improvements. Enable it and release it.

**Comment by Memo-Es** [comment]
@SudharakaP is this live?

**Comment by SudharakaP** [comment]
@Memo-Es : We haven't enabled it on prod yet. I was going to followup on this this week; I was wondering if we could enable this last week; https://opencollective.slack.com/archives/C03JQD12DU6/p1655387899818849 and since didn't receive any feedback thought of giving one more week before enabling it on prod. Will do so this week.  😉

**Comment by BenJam** [comment]
Closing as I think this is complete.
